src/commands/logs.ts

```diff
@@ -0,0 +1,32 @@
+// Output the logs of the bots
+
+import { CommandInteraction, PermissionFlagsBits, SlashCommandBuilder } from "discord.js";
+import type { Command } from "@/types.js";
+import fs from "node:fs";
+
+const LOG_PATH = "./bot-logs.txt";
+const TEMP_LOG_PATH = "./bot-logs-temp.txt";
```

Comment:
Just so you're aware, the logs will get cleared on restart due to a new clean container.